uktrade/data-hub-frontend
# Add investment project admin page

## Description of change

This PR allows users from the FDI verification team to move a project between stages in the front end rather than in Django admin. Users with the correct 'can_move_to_any_stage' permission will be able to see the 'Admin' tab on an investment project. On this page they will be able to select any other stage to move the project to. A user is then redirected to the project details page if the request is successful. 

This PR is linked to another ( #2675 ) which sets a flash message in session storage which is then retrieved by the FlashMessages react component. They have been separated for clarity. 

## Test instructions

- Ensure you have the `investment.change_to_any_stage_investmentproject` permission
- Go to an investment project and click on the 'Admin' tab
- Select a new stage or click save without selecting a stage to view validation error
- Check that the new stage is displayed on the project details page

## Screenshots

### After

![Screenshot 2020-05-14 at 14 16 12](https://user-images.githubusercontent.com/22460823/81938887-7ae77b80-95ed-11ea-8e07-88a171ee065e.png)

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
